### DISCUSSION TITLE
accessing raymath

### DISCUSSION THREAD

**Discussion by tgirod** [TSnake41/raylib-lua#23] [Q&A] [answered]
Hello! Can anyone explain to me how I can access functions that are defined in raymath, such as `Vector2Length`?

**Comment by TSnake41** [comment, marked as answer]
This function is available in `rl` table.
Like `rl.Vector2Length`

**Reply by tgirod** [reply]
shit I'm sorry for wasting your time and mine, I guess I typed `rl.Vector2length` or something like that.
